src/Criteo.OpenApi.Comparator.UTest/OpenApiSpecificationsCompareTests.cs

```diff
@@ -810,6 +810,25 @@ public void Compare_OAS_Should_Return_Nullable_Property_Changed_When_Nullable_Pr
             });
         }
 
+        [Test]
+        public void CompareOAS_ShouldReturn_ReferenceRedirectionAndTypeChangedDifferences()
+        {
+            var differences = CompareSpecifications("reference_redirection_type_changed");
+
+            differences.AssertContains(new ExpectedDifference
+            {
+                Rule = ComparisonRules.ReferenceRedirection,
+                Severity = Severity.Warning,
+                OldJsonRef = "#/paths/~1orders/get/responses/200/content/application~1json/schema/items"
+            }, 1);
+            differences.AssertContains(new ExpectedDifference
+            {
+                Rule = ComparisonRules.TypeChanged,
+                Severity = Severity.Warning,
+                OldJsonRef = "#/paths/~1orders/get/responses/200/content/application~1json/schema/items/properties/id/type"
```

Comment:
Can we assert the newJsonRef also? (I want to see that the new schema name if visible)

Reply:
Done